Method,Name,Error,Occurrences
GET,/supply/callcreatesupply,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))",20
GET,/supply/callcreatesupply,"HTTPConnectionPool(host='127.0.0.1', port=4000): Max retries exceeded with url: /supply/callcreatesupply (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f51e8062760>: Failed to establish a new connection: [Errno 111] Connection refused'))",2700
